perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -300,6 +295,7 @@ sub action_router : Path('') : Args(2) {
     my ( $self, $c, $id, $action ) = @_;
 
     $c->go( 'map', [ $id ] ) if $action eq 'map';
+    $c->go( 'nearby', [ $id ] ) if $action eq 'nearby';
```

Comment:
I realise we're not at all consistent here, but some indication this returned JSON might be useful; nearby.json? 'ajax' in the URL somewhere as some other calls do?